api/src/domain/license-status/searchLicenseStatusFactory.test.ts

```diff
@@ -43,9 +48,19 @@ describe("searchLicenseStatusFactory", () => {
     });
   });
 
-  describe("when license type is not public movers and warehouse", () => {
+  describe("when license type is health care services", () => {
```

Comment:
[dust] Should we consolidate public moves and health care services under a single describe referencing the logic associated with the dynamics API?

Reply:
done